modules/stanford_profile_styles/package-lock.json

```diff
@@ -4457,8 +4457,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
```

Comment:
You seem to have changes in every PR that is updating the package-lock.json file. What is it that you are doing that is causing this file to change? You should be using the nvm node version and running `npm install` only and that should leave this file untouched.

Reply:
@sherakama, 
Good question. 
I did:
`nvm use`
`npm install`
`npm rebuild node-sass`

Then, I've been running `npm run publish ; lando drush cr` to update the styles.
It appears that `npm run publish` updates the `package-lock.json`

Reply:
🤔 Hmmm, perhaps that has something to do with NPM's built-in `publish` command?

Reply:
I deleted the package-lock file. 
Thanks for pointing this out.